packages/paste-website/src/utils/componentFilters.ts

```diff
@@ -1,22 +1,10 @@
-import {PackageStatus} from '../constants';
-
 export interface ComponentNode {
   name: string;
   packageName?: string;
   version?: string;
   packageStatus?: string;
 }
 
-export const filteredComponents = (component: ComponentNode): boolean => {
-  return (
-    component.packageStatus !== PackageStatus.BACKLOG &&
```

Comment:
question: so we arent filtering these anymore?

Reply:
So Backlog status hasn't really been a thing since we moved the roadmap to airtable, and these 4 other "internal" packages don't need filtering as they aren't listed in the airtable as a consumer facing system package.

Reply:
We only had to filter these as the nav was build from workspace packages, which probably wasn't the best consumer facing API to use in hindsight

Reply:
Got it.